### PR TITLE
Bundled themes: bump versions for 6.3, with JSON

### PR DESCRIPTION
**Pull request by sabernhardt**
1. Updates theme version numbers in `style.css` and `readme` files, plus any package JSON files.
2. Sets "Tested up to" as 6.3.
3. Adds changelog links in `readme` files.
4. Updates version strings for assets modified recently.

https://core.trac.wordpress.org/ticket/57857

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by spacedmonkey** [comment]
While we are updating these, I would love to ensure that all of the readme.txt have `Requires at least` and `Requires PHP`.

File: src/wp-content/themes/twentytwelve/readme.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 === Twenty Twelve ===
 Contributors: wordpressdotorg
-Tested up to: 6.2
-Stable tag: 3.9
+Requires at least: WordPress 3.5
```
Comment (spacedmonkey) [inline src/wp-content/themes/twentytwelve/readme.txt]:
```suggestion
Requires at least: 3.5
```
WordPress here is not needed. It should be removed all places.

Comment (sabernhardt) [reply src/wp-content/themes/twentytwelve/readme.txt]: I apparently looked at the wrong theme for an example. "WordPress" should be removed now.

**Review by spacedmonkey — APPROVED** [review]
SGTM

**Comment by audrasjb** [comment]
committed in https://core.trac.wordpress.org/changeset/56315

**Pull request closed without merging**